src/deployment/deploy.py

```diff
@@ -569,12 +570,22 @@ def apply_migrations(self) -> None:
         migrate(table_service, self.migrations)
 
     def set_instance_config(self) -> None:
-        logger.info("setting instance config")
+        logger.info("setting instance configfdfdfdfd")
```

Comment:
configfdfdfdfd ?

Reply:
Time to fix now that you're looking for review?

Reply:
Fixed.